menu.c

```diff
@@ -479,8 +482,10 @@ void menu_redraw_motion(struct Menu *menu)
 
   if (C_ArrowCursor)
   {
-    /* clear the pointer */
-    mutt_window_addstr("  ");
+    /* clear the arrow */
+    size_t b = mutt_strwidth(C_ArrowString);
+    for (int p = 0; p <= b; p++) //mutt_window_addstr() would be more pleasant here
```

Comment:
I can change `mutt_window_addch()` and entire for loop to:
```
mutt_window_printf("%*s", b+1, "");
```

Reply:
or even save spare variable which i didn't know how to name…
```
mutt_window_printf("%*s", mutt_strwidth(C_ArrowString) + 1, "");
```

Reply:
That's nice use of `printf()` however not everyone will be familiar with the syntax.
I suggest a comment:
```
// Print space chars to match the screen width of C_ArrowString
```